src/renderer/features/fellowship-overview/components/MembersTab.tsx

```diff
@@ -0,0 +1,16 @@
+/* eslint-disable i18next/no-literal-string */
+import { FootnoteText, TitleText } from '@/shared/ui/Typography';
+import { Box } from '@/shared/ui-kit';
+
+export const MembersTab = () => {
+  return (
+    <div className="p-5">
+      <Box gap={6}>
+        <Box gap={1}>
+          <TitleText className="text-text-primary">Members</TitleText>
+          <FootnoteText className="text-text-primary">Coming soon</FootnoteText>
```

Comment:
text-text-primary is the default class for these components
```suggestion
<TitleText>Members</TitleText>
<FootnoteText>Coming soon</FootnoteText>
```

Reply:
Move strings to the i18n `en.json`

Reply:
Fixed